### DISCUSSION TITLE
publishing a message across two super-clusters with two different operators via gateways

### DISCUSSION THREAD

**Discussion by skaluva** [nats-io/nats.java#1097] [General]
Can we form supercluster with 2 or more clusters operates with different operators if we configure with nsc? if so how can we send a message from one cluster to another through  gateway.

We tried at our end, but it didnt work.  We also gave permissions one account to another account which are in different operators.
But we are getting below errror:

**_can't locate account [xyz] for import of [testsubject]_**

**Comment by scottf** [comment]
@skaluva This isn't so much a client issue, it has to do with permissions and possibly configuring the domain or prefix in the JetStream context. Could you please ask this question in slack?
